test/python/providers/test_fake_backends.py

```diff
@@ -256,8 +260,8 @@ def test_converter_delay_circuit(self):
 
     def test_converter_with_missing_gate_property(self):
         """Test converting to V2 model with irregular backend data."""
-        backend = FakeOpenPulse2Q()
-
+        with self.assertWarns(DeprecationWarning):
+            backend = FakeOpenPulse2Q()
```

Comment:
Hmm... not sure... I assumed that it will be removed with pulse. 

Is there a possibility of a gate in BackendV2 with calibration but no "property"?

```python
        # The backend includes pulse calibration definition for U2, but its property is gone.
        # Note that u2 is a basis gate of this device.
        # Since gate property is not provided, the gate broadcasts to all qubits as ideal instruction.
        del backend._properties._gates["u2"]
```

Honestly, I think I'm missing what this test is testing. @nkanazawa1989 ? @mtreinish ?

Reply:
My comment might have been confusing, it's not necessarily about this test. I was thinking of the general context of the `BackendV1` deprecation (#12629) and whether we should also deprecate `BackendConverter` with it. I find it tricky because `BackendConverter` is part of the migration path, so it's strange to have it raise a deprecation warning when we tell people to use it, but it cannot be tested once we get rid of `BackendV1`, so in that sense I think it should be deprecated. We can maybe raise a custom warning to explain this situation with it.